efd/salkafka/setup.sh

```diff
@@ -1,28 +1,34 @@
 #!/usr/bin/env bash
+#
```

Comment:
Another big commented block. Can it just be removed?

Reply:
We are still at a point where I need both deploy and develop to build some of the containers. I'd like to keep the comment here at least until next round of deployments.

Reply:
That's fine by me. Thanks for the update.